listener.py

```
@@ -47,52 +47,109 @@ def build_client() -> tweepy.Client:
+                    "body": f"Intended reply:\n{reply.text}\n\nError: {exc}",
+                    "actions": [],
+                    "metadata": {
+                        "agent_id": member.profile.id,
+                        "mention_id": mention.id,
+                        "error": str(exc),
+                    },
+                },
+                posted_by=member.profile.id,
+            )
+        except Exception:
+            pass
```

Ruff (0.15.20)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)